client/state/selectors/test/get-magic-login-requested-auth-successfully.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getMagicLoginRequestedAuthSuccessfully } from '../';
+
+describe( 'getMagicLoginRequestedAuthSuccessfully()', () => {
+	it( 'should return false if there is no information yet', () => {
+		const status = getMagicLoginRequestedAuthSuccessfully( undefined );
+		expect( status ).to.be.false;
+	} );
+
+	it( 'should return true if true', () => {
```

Comment:
Would be better to describe it as something like `should return true if requested auth succeeded`